tools/github_readme_sync/tests/readme_test.py

```diff
@@ -511,6 +511,23 @@ def test_convert_cloudinary_videos(self):
             block_str = f"[block:html]\n{json.dumps(block, indent=2)}\n[/block]"
             self.assertIn(block_str, result)
 
+    def test_caption_markdown_images_multiple_per_line(self):
```

Comment:
Should this be part of this PR: https://github.com/thousandbrainsproject/tbp.monty/pull/64